jest.setup.ts

```diff
@@ -0,0 +1 @@
+import '@testing-library/jest-dom/extend-expect'
```

Comment:
you should load the mock environment variables here too

Reply:
Let's deal with API testing and mocking APIs generally in another branch